mne/minimum_norm/inverse.py

```diff
@@ -1449,31 +1452,34 @@ def make_inverse_operator(info, forward, noise_cov, loose='auto', depth=0.8,
-        +---------------------+-----------+-----------+-----------+-----------------+--------------+
-        | | Free orientation  | 1.0       | None      | False     | False           | True | False |
-        +---------------------+-----------+-----------+-----------+-----------------+--------------+
-        | | Fixed constraint, | 0.0       | 0.8       | True      | False           | True         |
-        | | Depth weighted    |           |           |           |                 |              |
-        +---------------------+-----------+-----------+-----------+-----------------+--------------+
-        | | Fixed constraint  | 0.0       | None      | True      | True            | True         |
-        +---------------------+-----------+-----------+-----------+-----------------+--------------+
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | Inverse desired                             | Forward parameters allowed  |
+        +=====================+===========+===========+===========+=================+
+        |                     | **loose** | **depth** | **fixed** | **force_fixed** |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Loose constraint, | 0.2       | 0.8       | False     | False           |
+        | | Depth weighted    |           |           |           |                 |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Loose constraint  | 0.2       | None      | False     | False           |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Free orientation, | 1.0       | 0.8       | False     | False           |
+        | | Depth weighted    |           |           |           |                 |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Free orientation  | 1.0       | None      | False     | False           |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Fixed constraint, | 0.0       | 0.8       | True      | False           |
+        | | Depth weighted    |           |           |           |                 |
+        +---------------------+-----------+-----------+-----------+-----------------+
+        | | Fixed constraint  | 0.0       | None      | True      | True            |
+        +---------------------+-----------+-----------+-----------+-----------------+
 
     Also note that, if the source space (as stored in the forward solution)
     has patch statistics computed, these are used to improve the depth
     weighting. Thus slightly different results are to be expected with
     and without this information.
+
+    .. versionchanged:: 0.20
+       Surface orientation is always used to ensure consistency.
```

Comment:
... and this isn't currently correct, though maybe we should change the behavior this way.